t/prototype.t

```diff
@@ -38,3 +38,16 @@ is( prototype('f_decl'), '$$$$', 'forward declaration');
 
 set_prototype('\%', \&f_decl);
 is( prototype('f_decl'), '\%', 'change forward declaration');
+
+{
+  my $warning;
+  local $SIG{__WARN__} = sub { $warning = shift };
+
+  my @c = prototype();
+  is( scalar @c, 0, 'no arg results in empty list'); # XXX
```

Comment:
Inconsistent whitespacing - probably wants to be

    is(scalar @c, 0, 'no arg results in empty list'); # XXX